src/main/java/core/basesyntax/strategy/StorageStrategy.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.impl.BalanceStorageService;
+import core.basesyntax.strategy.impl.PurchaseStorageService;
+import core.basesyntax.strategy.impl.ReturnStorageService;
+import core.basesyntax.strategy.impl.SupplyStorageService;
+
+public class StorageStrategy {
+    public StorageService getStorageService(FruitTransaction transaction) {
+        switch (transaction.getOperation().toString()) {
+            case "SUPPLY":
+                return new SupplyStorageService();
+            case "PURCHASE":
+                return new PurchaseStorageService();
+            case "RETURN":
+                return new ReturnStorageService();
+            case "BALANCE":
+            default:
+                return new BalanceStorageService();
```

Comment:
what if new Operation is added and this method is not updated
better by default to throw some custom exception ex.: UnsupportedOperationException